app/services/current-user.js

```diff
@@ -9,7 +9,10 @@ export default Service.extend({
     if (this.get('session.isAuthenticated')) {
       let user = this.get('store').queryRecord('user', {me: true});
       this.set('user', user);
-      user.catch(() => {
+      return user.then((user) => {
+        this.set('user', user);
```

Comment:
Yeah I think this was leftover from a weird issue when we were doing that weird double authentication thing and can be removed now. I'll clean it up and make sure it still works.